src/components/Metrics/index.js

```diff
@@ -0,0 +1,94 @@
+import { RichTypography } from "@commons-ui/core";
+import { IconButton, Grid, useMediaQuery } from "@material-ui/core";
+import { useTheme } from "@material-ui/core/styles";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+import DataVisualCard from "@/pesayetu/components/DataVisualCard";
+import Section from "@/pesayetu/components/Section";
+
+function Metrics({ items, title, ...props }) {
+  const classes = useStyles(props);
+  const theme = useTheme();
+  const isTablet = useMediaQuery(theme.breakpoints.only("md"));
+  if (!items?.length) {
+    return null;
+  }
+  return (
+    <div className={classes.root}>
+      <Section classes={{ root: classes.section }}>
+        <RichTypography variant="h2" className={classes.sectionTitle}>
+          {title}
+        </RichTypography>
+        {items?.map((item, index) => (
+          <Grid
+            key={item.title}
+            container
+            direction={index % 2 === 0 ? "row" : "row-reverse"}
+            justifyContent="space-between"
+            alignItems="flex-start"
+            className={classes.metrics}
+          >
+            <Grid item xs={12} md={5} lg={3}>
+              <Grid container item direction="row" alignItems="center">
+                <IconButton color="primary" size="small">
+                  <Image src={item.icon} width={44} height={44} />
+                </IconButton>
+                <RichTypography variant="h4" className={classes.title}>
+                  {item.title}
+                </RichTypography>
+              </Grid>
+              <RichTypography variant="body2" className={classes.description}>
+                {item.description}
+              </RichTypography>
+            </Grid>
+            <Grid
+              item
+              xs={12}
+              md={7}
+              lg={8}
+              container
+              direction="row"
+              justifyContent={
+                isTablet && index % 2 === 0 ? "flex-end" : "flex-start"
+              }
+              alignItems="center"
```

Comment:
This is for alignment issues ....if we only use flex-start, especially for tablet we get this :pointdown
Screenshot 2021-08-24 at 10.39.45

Reply:
This was for the tablet layout..since we are row reversing the div in the the second grid the justifyContent will also have to be different in the second grid. If we don't do this we get 👇🏽 

<img width="578" alt="Screenshot 2021-08-24 at 10 39 45" src="https://user-images.githubusercontent.com/89015352/130583274-d352e1c0-6cb8-47bf-9c75-b7def2c795ca.png">

Reply:
I don't think that's the reason @nerdlet ... I believe [this](https://github.com/CodeForAfrica/PesaYetu/blob/6aa1269e72bbff1ee6a25fd8e106e933c49af379/src/components/DataVisualCard/useStyles.js#L7-L13) is where the issues starts. in tablet & desktop, `maxWidth is` set and we'd need to override that as well. As long as the parent container has `justifyContent="space-between"`, `row` and `row-reverse` should behave exactly the same i.e. the two items will be pushed as far apart as possible.

Reply:
Okay...will make the changes

Reply:
@kilemensi Done